lib/Stats.tsx

```diff
@@ -1,101 +1,236 @@
-import React, { useMemo } from 'react'
+import React, { useCallback, useEffect, useRef, useState } from 'react'
 import styled from 'styled-components'
 import { VapixParameters, VideoProperties } from './PlaybackArea'
 
+import { StreamStats } from './img'
+
 const StatsWrapper = styled.div`
-  background: rgb(0, 0, 0, 0.66);
   position: absolute;
   top: 24px;
   left: 24px;
-  padding: 8px 16px;
-  font-family: sans-serif;
+  width: 360px;
+  min-width: 240px;
   max-width: 80%;
   max-height: 80%;
+  border-radius: 4px;
+  background: #292929 0% 0% no-repeat padding-box;
+  opacity: 0.88;
+  overflow: scroll;
 `
 
 const StatsHeader = styled.div`
-  color: white;
+  padding: 8px 24px;
+  border-bottom: 1px solid #525252;
+`
+
+const StatsIcon = styled.span<{ readonly clickable: boolean }>`
+  width: 24px;
+  height: 24px;
+  float: left;
+  cursor: ${({ clickable }) => (clickable ? 'pointer' : 'default')};
+
+  & > svg {
+    fill: #e0e0e0;
+  }
+`
+
+const StatsTitle = styled.span`
+  display: inline-block;
+  margin-left: 8px;
+  vertical-align: sub;
   text-align: left;
-  font-size: 14px;
-  font-weight: bold;
+  font-size: 16px;
+  font-family: 'Open Sans';
```

Comment:
Should probably include a fallback as well. Like sans-serif or something.

Reply:
Yes, agree, use a sans serif fallback (serif looks a bit odd).

Reply:
Done